src/index.js

```diff
@@ -34,22 +33,28 @@ class EventList extends React.Component {
     this.state = {
       events: [],
       isLoading: true,
-      error: null
+      error: null,
+      number: 0
     }
     getUpcomingEvents().then(data => {
       this.setState({
         events: data[props.eventGroup].slice(0, 10),
-        isLoading: false
+        isLoading: false,
+        number: props.eventGroup === 'intern' ? 2 : 4
```

Comment:
Rename til et mer fornuftig navn. Måtte grave i resten av koden for å forstå hva dette var ment som